### PR TITLE
Bb prediction cookbook recipe

### PR DESCRIPTION
**Pull request by beczkowb**

### CODE REVIEW

File: docs/ml-prediction.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+prediction environment you need, and make predictions more efficient using GPU.
+
+BigFlow 1.3 comes with utilities that make your life easier when it comes to writing Dockerized
+processes and writing prediction logic.
+
+This recipe will show you how to set up a PyTorch prediction process, but the dockerization allows you
+to define an environment for any framework. We assume that you are already familiar with the BigFlow API.
+
+We are going to create a batch prediction process for the Fashion MNIST model, using Dataflow. You can download
+the model here. We won't focus on the utility or performance side of the project, but more on the API that
```
Comment (iwko) [inline docs/ml-prediction.md:21]: here? link to model?

File: docs/ml-prediction.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+# use it if you want ot use custom Docker image on Dataflow workers
+FROM apache/beam_python3.7_sdk:2.27.0
+
+
+# Cherry-pick apache beam binaries from offical image to our custom.
+# Use this only in case when you unable to use original image (FROM apache/beam...)
+# COPY --from=apache/beam_python3.6_sdk:2.26.0 /opt/apache/beam /opt/apache/beam
+
+
+# Uncomment if you want to install CUDA drivers to your docker
```
Comment (iwko) [inline docs/ml-prediction.md:39]: We should reverse this comment imo. `Comment if you do not want to install CUDA drivers...`

File: docs/ml-prediction.md (hunk 1 above)
Comment (beczkowb) [reply docs/ml-prediction.md:18]: done

File: docs/ml-prediction.md (hunk 2 above)
Comment (beczkowb) [reply docs/ml-prediction.md]: done, kinda

**Review by iwko — APPROVED** [review]

**Pull request merged**